src/app/pages/ArticlePage/ArticleHeadline/index.tsx

```diff
@@ -48,19 +34,9 @@ const ArticleHeadline = (props: ComponentToRenderProps) => {
       />
       {showLiteCTAOnCanonical && (
         <>
+          <div css={styles.liteCTAContainer} data-e2e="to-lite-site-loading" />
```

Comment:
Minor but looks like this `data-e2e` can be removed? Unless I've missed it being referenced!

Reply:
Yup I think it can now be removed!